index.html

```diff
@@ -242,207 +229,153 @@ <h2 class="primary-text">Multi-Post Stories</h2>
-                <img
-                  class="language-list-item-image"
-                  src="./assets/images/css.png"
-                  alt="Javascript image"
-                />
-                <p class="language-list-item-text">CSS</p>
-              </div>
+          </li>
+          <li>
+            <a href="https://angel.com/">
+              <img src="./assets/images/angle.png" alt="Angle image" />
             </a>
-          </ul>
-        </li>
-        <li class="drop-options-list">
-          <ul class="hidden-options-title">
-            <li>Frameworks</li>
-            <li>
-              <svg
-                width="24"
-                height="25"
-                viewBox="0 0 24 25"
-                fill="none"
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <path
-                  fill-rule="evenodd"
-                  clip-rule="evenodd"
-                  d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z"
-                  fill="#505F79"
-                />
-              </svg>
-            </li>
-          </ul>
-          <hr />
-        </li>
-        <li class="drop-options-list">
-          <ul class="hidden-options-title">
-            <li>Skills</li>
-            <li>
-              <svg
-                width="24"
-                height="25"
-                viewBox="0 0 24 25"
-                fill="none"
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <path
-                  fill-rule="evenodd"
-                  clip-rule="evenodd"
-                  d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z"
-                  fill="#505F79"
-                />
-              </svg>
-            </li>
-          </ul>
-          <hr />
-        </li>
-      </ul>
+          </li>
+          <li>
+            <a href="https://twitter.com/">
+              <img src="./assets/images/twitter.png" alt="twitter image" />
+            </a>
+          </li>
+          <li>
+            <a href="">
+              <img src="./assets/images/medium.png" alt="Medium image" />
+            </a>
+          </li>
+        </ul>
+        <button type="button" class="btn">Get my resume</button>
+      </div>
+      <div class="about-right">
+        <ul class="drop-options">
+          <li class="drop-options-list">
+            <ul class="hidden-options-title">
+              <li>Languages</li>
```

Comment:
:red_circle: Please make sure you pick the right font color from the Figma template. As for now, Your font color does not reflect the font color in the Figma Design

Your Design
![fontcolorstudent](https://user-images.githubusercontent.com/26736582/199822096-efeacab7-44da-44f0-84e8-4f3d743e2698.png)


Figma Design

![fontcolor](https://user-images.githubusercontent.com/26736582/199822021-6874e374-5f77-47aa-af5f-60838408cb86.png)